### PR TITLE
Updates to character count documentation

### PR DESCRIPTION
**Pull request by stevenaproctor**
This is following the Design System Working Group and a conversation with @amyhupe.

### CODE REVIEW

File: src/components/character-count/index.md.njk [hunk 1]
```diff
@@ -80,37 +76,37 @@ Error messages should be styled like this:
...
 
-Say “[whatever it is] must be [number] characters or less”.<br>
-For example, “Summary must be 400 characters or less”.
+Say ‘[whatever it is] must be [number] characters or less’.<br>
+For example, ‘Summary must be 400 characters or less’.
 
 ## Research on this component
 
-This component was originally developed and tested by the Government Digital Service (GDS) in a prototype of the Apply for a temporary event notice service.
+This Government Digital Service (GDS) developed and tested this component in a prototype of the ‘Apply for a temporary event notice’ service.
```
Comment (36degrees) [inline src/components/character-count/index.md.njk]: Not sure this is quite right - "This Government Digital Service (GDS) developed"

Comment (amyhupe) [inline src/components/character-count/index.md.njk]:
```suggestion
The Government Digital Service (GDS) developed and tested this component in a prototype of the ‘Apply for a temporary event notice’ service.
```

Comment (amyhupe) [reply src/components/character-count/index.md.njk]: That's all from me. If you can update this, I'll approve.

Comment (stevenaproctor) [reply src/components/character-count/index.md.njk]: My mistake. In a rush to update it.

Comment (stevenaproctor) [reply src/components/character-count/index.md.njk]: Done.

**Pull request merged**